src/Composer/Autoload/AutoloadGenerator.php

```diff
@@ -256,14 +257,24 @@ public static function autoload(\$class)
                             continue;
                         }
 
-                        $classMap = $this->addClassMapCode($filesystem, $basePath, $vendorPath, $dir, $blacklist, $namespace, $group['type'], $classMap);
+                        $classMap = $this->addClassMapCode($filesystem, $basePath, $vendorPath, $dir, $blacklist, $namespace, $group['type'], $classMap, $ambigiousClasses);
                     }
                 }
             }
         }
 
         foreach ($autoloads['classmap'] as $dir) {
-            $classMap = $this->addClassMapCode($filesystem, $basePath, $vendorPath, $dir, $blacklist, null, null, $classMap);
+            $classMap = $this->addClassMapCode($filesystem, $basePath, $vendorPath, $dir, $blacklist, null, null, $classMap, $ambigiousClasses);
+        }
+
+        foreach($ambigiousClasses as $className => $ambigiousPaths)
+        {
```

Comment:
Should be

```suggestion
        foreach ($ambigiousClasses as $className => $ambigiousPaths) {
```

To follow CS